tests/integration/pilot/multi_version_revision_test.go

```diff
@@ -199,4 +199,8 @@ func skipIfK8sVersionUnsupported(t framework.TestContext) {
 	if !t.Clusters().Default().MinKubeVersion(16) {
 		t.Skipf("k8s version not supported for %s (<%s)", t.Name(), "1.16")
 	}
+	// Kubernetes 1.22 drops support for a number of legacy resources, so we cannot install the old versions
+	if !t.Clusters().Default().MaxKubeVersion(21) {
```

Comment:
!t.Clusters().Default().MaxKubeVersion(21) checks the minor k8s version is at most 21 but does not check the major k8s version. If the k8s version is 2.1 (major-version=2, minor-version=1), what is the behavior of !t.Clusters().Default().MaxKubeVersion(21)?

Reply:
Kubernetes will never have a 2.0 according to their project. If they do, we can change it.